### DISCUSSION TITLE
Will you be looking to support other content types such as application/scim+json

### DISCUSSION THREAD

**Discussion by russ-milburn** [mockoon/mockoon#1580] [Ideas]

**Comment by 255kb** [comment]
Not sure what you are referring to. 
Mockoon [supports request's body parsing for all `application/*+json` Content-Type](https://mockoon.com/docs/latest/requests/supported-body-formats/#json-support). Which mean you can use this Content-Type and use the body's properties in templating helpers and rules. 

`application/scim+json` is JSON in the end. If you mean does Mockoon supports this specific schema, the answer is no. There is no built-in support for any kind of schema. However, we recently added JSON schema validation in the rules. You will find more information [in the docs](https://mockoon.com/docs/latest/route-responses/dynamic-rules/#json-schemas) regarding this feature.

**Comment by russ-milburn** [comment]
thank you for your reply, application/scim+json is a system for cross domain identity management. I have noticed that if i use this as a content type when i send a request to mockoon i am unable to retrieve anything from the body of the request. I noticed that this was not a supported type so i was wondering if you were planning to support more content types.

**Reply by 255kb** [reply]
As I said, it's not explicitely a listed supported type but we support all Content-Type with the following regex `/application\/.*\+json/` which includes `application/scim+json`.
I just tested, if I send the following GET request: 

```text
GET /test
Content-Type: application/scim+json

{
  "test": "value"
}
```

With the following response body: `{{bodyRaw 'test'}}`.
I get the following response: "value"

**Comment by russ-milburn** [comment]
sorry my bad, I looked at my client and I was running version 3.0 of Mockoon desktop. I can see this is working perfectly now.

thank you for the assist

**Reply by 255kb** [reply]
That's an old version indeed! Glad it worked :)
